packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -394,6 +422,9 @@ class ScrollableBaseNative extends Component {
 			}
 		}
 
+		this.forwardReachEdgeEvent('horizontal');
+		this.forwardReachEdgeEvent('vertical');
```

Comment:
If `clientSize` prop is passed, then `componentDidMount` will be called, but `componentDidUpdate` will not be called. Please check this edge case.